content/00.front-matter.md

```diff
@@ -0,0 +1,12 @@
+# Manubot Rootstock: Manuscript Title
+
+## Authors
+
+{% for author in authors %}
++ **{{author.full_name}}**<br>
+  [\@{{author.github_username}}](https://github.com/{{author.github_username}}) ·
+  [{{author.orcid}}](https://orcid.org/{{author.orcid}}) ·
+  {{author.affiliations}} ·
+  Funded by {{author.funders}}
```

Comment:
One possibility would be to put this author section at the end of the manuscript and then just include an author list in the beginning. Leaving this decision to @evencofer if he submits a PR to address the author metadata as per https://github.com/greenelab/manubot-rootstock/issues/4#issuecomment-313883585.